scripts/fledge

```diff
@@ -805,14 +811,17 @@ get_fledge_version
 get_rest_api_url
 
 # Call getopt to get any command line options for username and password
-while getopts "u:p:" option; do
+while getopts "u:p:c:" option; do
```

Comment:
We can add the `-h` option to display the help usage, showing all possible options for stopping the FL instance.

Reply:
We already have fledge help, so the information is available. True that should be enhanced to cover the -u, -p and -c arguments.